srcpkgs/chronograf/template

```diff
@@ -18,6 +18,7 @@ system_accounts="_chronograf"
 _chronograf_homedir="/var/lib/${pkgname}"
 make_dirs="${_chronograf_homedir} 0755 _chronograf _chronograf"
 
+broken="fails to build v8: error: 'class Nan::Persistent<v8::Object>' has no member named 'IsNearDeath'"
```

Comment:
Might be worth just switching to upstream assets if they vendor a tarball.

Reply:
Not sure this one is necessary anymore, since you merged the fix.